database/migrations/2020_06_16_215943_create_mentee_profiles_table.php

```diff
@@ -25,7 +25,7 @@ public function up()
             $table->text('timeframe')->nullable();
             $table->text('suitable_time')->nullable();
             $table->text('extra_info')->nullable();
-            $table->tinyInteger('status')->unsigned()->default(ProfileStatus::Pending);
+            $table->tinyInteger('status')->unsigned()->default(ProfileStatus::Confirmed);
```

Comment:
@fourstacks will this cause any issues when deploying to the existing environment that you set up on laravel forge? I edited an existing migration because when I tried to create a new migration to change the default for this column, it didn't seem to like me changing a tinyint column.... I think it's because [the docs](https://laravel.com/docs/8.x/migrations#modifying-columns) say this:
> Only the following column types can be "changed": bigInteger, binary, boolean, date, dateTime, dateTimeTz, decimal, integer, json, longText, mediumText, smallInteger, string, text, time, unsignedBigInteger, unsignedInteger, unsignedSmallInteger and uuid.

Reply:
Sorry for slow reply!  Not too sure if the site as it stands on Forge has any actual data in there.  If not then no big deal - we'll just blow away the database and remigrate.  If there is data we can just fiddle the default value directly on the DB itself